docs/pages/contributing/documentation.mdx

```diff
@@ -24,16 +24,14 @@ the kind of documentation we want to transform our current documentation into.
 
 </Admonition>
 
-- [How to Contribute to Teleport's
-  Documentation](./documentation/how-to-contribute.mdx): Follow this guide to
-  start contributing changes to Teleport's documentation.
-- [How to Review Documentation Changes](./documentation/reviewing-docs.mdx):
-  Read our recommended approach to preview docs changes in order to review pull
+- [How to contribute to Teleport documentation](./documentation/how-to-contribute.mdx) describes 
```

Comment:
This may be a matter of taste, but "Teleport's documentation" sounds more natural to me for some reason. What is the reasoning behind using "Teleport documentation"?

Reply:
I think of Teleport as the modifier here, that is, Teleport is what the documentation is about, not who it belongs to. Typically, you don't use the possessive form with company or product names. I think there's some legal reason that's this is considered modifying the brand name.

Reply:
What would you think of "the Teleport documentation"? I.e., it's a discrete entity and not a mass noun or type of thing?